### ISSUE TITLE
WSL support?

### ISSUE THREAD

**Issue by eliw00d** [trixnz/vscode-lua#31]
I have luacheck installed in WSL and it can be run from Windows using `bash -c "luacheck"` (see [here](https://msdn.microsoft.com/en-us/commandline/wsl/interop)).

I tried setting `lua.luacheckPath` to `"bash -c \"luacheck\""` in my settings, but this does not seem to work.

Any thoughts on best way to support this?

**Comment by eliw00d** [comment]
Any word?

**Comment by eliw00d** [comment]
@trixnz It has been awhile since I worked with Lua but I recently started up again and switched to WSL 2. I followed [the recommended file storage setup](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#file-storage) and since it is now actually in the Linux subsystem it worked using the Linux path to luacheck: 

```json
	"lua.luacheckPath": "/usr/local/bin/luacheck",
```

Just in case anyone else ran into this in the past. That means we can close this as a non-issue (as of WSL 2).
